site/en/blog/deps-rems-101/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Deprecations and removals in Chrome 101
+description: >
+  A round up of the deprecations and removals in Chrome 101 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2022-03-31
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/GLywh0GpxQEByxtmCaVY.png'
+alt: >
+  Deprecations and Removals hero logo
+tags:
+  - deprecations
+  - removals
+  - chrome-101
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 101 beta was released on March 31, 2022 and is expected to become the
+stable version in late April, 2022.
+
+## Reduce user agent string information
+
+Chrome is trying to [reduce the amount of information the user agent string exposes](https://www.chromestatus.com/feature/5704553745874944) in HTTP requests as well as in navigator.userAgent, navigator.appVersion, and navigator.platform. We're doing this to prevent the user agent string from being used for passive user fingerprinting. To join the origin trial, see [its entry on Chrome Origin Trials](/origintrials/#/view_trial/-7123568710593282047).
```

Comment:
```suggestion
Chrome is [reducing the amount of information the User-Agent string exposes](https://www.chromestatus.com/feature/5704553745874944) in HTTP requests as well as in navigator.userAgent, navigator.appVersion, and navigator.platform. We're doing this to prevent the user agent string from being used for passive user fingerprinting. To join the origin trial, see [its entry on Chrome Origin Trials](/origintrials/#/view_trial/-7123568710593282047).
```

Reply:
This is actively in process, rather than an attempt.

For Privacy Sandbox, when referencing the header or the string, it should be spelled "User-Agent," i.e. "User-Agent string" or "User-Agent header".  When referencing the software agent supporting the network, we spell it as "user agent".